### PR TITLE
[Feature Fix] [Style Guide] Fix bugs on Trellos

### PR DESCRIPTION
**Pull request by haoyuchen1992**
Resolved Trello Bugs:
1. https://trello.com/c/1UJYpqXd/21-success-alerts-should-be-green-not-yellow
2. https://trello.com/c/BgmyRw7d/35-no-close-button-for-info-modals

### CODE REVIEW

File: framework/auth/views.py [hunk 1]
```diff
@@ -88,11 +88,10 @@ def forgot_password_post():
                 reset_link=reset_link
             )
         status.push_status_message(
-            'An email with instructions on how to reset the password for the '
-            'account associated with {0} has been sent. If you do not receive '
-            'an email and believe you should have please '
-            'contact OSF Support.'.format(email)
-        )
+            ('An email with instructions on how to reset the password' +
```
Comment (sloria) [inline framework/auth/views.py]: You are missing spaces at the ends of these words.

Also, the `+` signs are unnecessary. See https://cosdev.readthedocs.org/en/latest/style_guides/python.html#line-lengths . 

The previous formatting of this message was correct; can you revert this change and just add the `'success'` argument?

File: website/project/views/register.py [hunk 2]
```diff
@@ -472,7 +472,10 @@ def node_register_template_page_post(auth, node, **kwargs):
         for child in register.get_descendants_recursive(lambda n: n.primary):
             child.set_privacy('public', auth, log=False)
 
-    push_status_message('Files are being copied to the newly created registration, and you will receive an email notification containing a link to the registration when the copying is finished.')
+    push_status_message((
+        'Files are being copied to the newly created registration, ' +
+        'and you will receive an email notification containing a link' +
```
Comment (sloria) [inline website/project/views/register.py]: `+` signs unnecessary here, too.

**Comment by sloria** [comment]
Pass finished. :point_up:

**Comment by haoyuchen1992** [comment]
@sloria Change has been made for code review!
